packages/OcLaunch/OcLaunch.0.1.3/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+name: "OcLaunch"
+version: "0.1.3"
+maintainer: "Joly Clément <leowzukw@vmail.me>"
+authors: [ "Joly Clément <leowzukw@vmail.me>" ]
+license: "CeCILL"
+homepage: "http://www.oclaunch.tuxfamily.org"
+build: [
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+]
+install: ["ocaml" "setup.ml" "-install"]
+depends: [
+  "ocamlbuild"
```

Comment:
There's no need for this dependency, since it is builtin to the OCaml compiler.  Try running `opam lint` to see that it's a missing dependency.  As a consequence of including it, the package becomes uninstallable in Travis:

```
The following dependencies couldn't be met:
  - OcLaunch -> ocamlbuild
```